### PR TITLE
Update "other" icon, fix blocks file uploading, trap BPC terminal echo

### PR DESCRIPTION
**Pull request by MatzElectronics**

### CODE REVIEW

**Comment by MatzElectronics** [comment]
Addresses #1064 #1077

**Comment by zfi** [comment]
@MatzElectronics: What is commit 0d116b0 UI FIxes actually fixing? It's a really old commit so I'm just wondering if it is actually correct.

**Comment by MatzElectronics** [comment]
It removes both spell checking and the button that allows people to put
URLs in their project descriptions.

File: src/main/webapp/cdn/project.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ var projectTypes = {
 var simplemde = null;
 
 $(document).ready(function () {
-    simplemde = new SimpleMDE({element: document.getElementById("project-form-description")});
+    simplemde = new SimpleMDE({element: document.getElementById("project-form-description"), hideIcons: ["link"], spellChecker: false});
```
Comment (zfi) [inline src/main/webapp/cdn/project.js]: Removes spell checking and the button that allows users to put URLs in their project descriptions.

**Pull request merged**